env.sh

```diff
@@ -45,6 +45,20 @@ action() {
         run_cmd scramv1 project CMSSW $CMSSW_VER
         run_cmd cd $CMSSW_VER/src
         run_cmd eval `scramv1 runtime -sh`
+        run_cmd git-cms-init
+        run_cmd git clone ssh://git@gitlab.cern.ch:7999/akhukhun/roccor.git RoccoR
+        run_cmd git clone git@github.com:SVfit/ClassicSVfit.git TauAnalysis/ClassicSVfit -b fastMTT_19_02_2019
+        run_cmd git clone git@github.com:SVfit/SVfitTF.git TauAnalysis/SVfitTF
+        run_cmd git clone ssh://git@gitlab.cern.ch:7999/rgerosa/particlenetstudiesrun2.git ParticleNetStudiesRun2 -b cmssw_124X
+        run_cmd mkdir -p $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoBTag/Combined/data/
+        run_cmd scp -r ParticleNetStudiesRun2/TrainingNtupleMakerAK4/data/ParticleNetAK4 $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoBTag/Combined/data/
+        run_cmd scp -r ParticleNetStudiesRun2/TrainingNtupleMakerAK8/data/ParticleNetAK8 $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoBTag/Combined/data/
+        run_cmd mkdir -p $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoTauTag/data/TauES/
+        run_cmd scp -r ParticleNetStudiesRun2/TrainingNtupleMakerAK4/data/TauES/* $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoTauTag/data/TauES/
+        run_cmd git cms-addpkg RecoBTag/Combined
+        run_cmd git cms-addpkg RecoTauTag
+        run_cmd cp -r $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoBTag/Combined/data RecoBTag/Combined/data
+        run_cmd cp -r $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoTauTag/data RecoTauTag/data
```

Comment:
why the same data should be copied into two places? one should be enough.

Reply:
It's required in this place, otherwise the files aren't propagated to crab job. For the previous place, it's needed to run locally. I don't know if it'll also work locally with the files in the second place; again to be checked